src/index.html

```diff
@@ -6,6 +6,18 @@
   </head>
   <body>
     <div id="root"></div>
+    <script src="https://www.gstatic.com/firebasejs/3.9.0/firebase.js"></script>
+<script>
+  var config = {
```

Comment:
Y u put code in scripts? 

Y u import firebase with `<script>` instead of `yarn add`?